enterprise/install-scripts/openshift.ks

```diff
@@ -1248,7 +1258,7 @@ YUM
 configure_cart_repos()
 { # add cartridge (or dependency) repo as needed
   local -A url=(
-          [jbosseap]="${jboss_repo_base}/jbeap/6/os"
+          [jbosseap]="${jboss_repo_base}/jbeap/${jbosseap_version:-6}/os"
```

Comment:
Why is the default value here 6 instead of 6.3 as set in `set_defaults()`? Why not just rely on `set_defaults()` to set a value and use `${jbosseap_version}` here?